lib/parsers/tiff_parser.rb

```diff
@@ -43,5 +46,9 @@ def cr2?(io)
     safe_read(io, 2) == 'CR'
   end
 
+  def arw?(exif_data)
+    exif_data.compression == 6 && exif_data.new_subfile_type == 1 && exif_data.make == 'SONY'
```

Comment:
Is this an exhaustive condition? Or there still is the possibility to miss a few ARWs (or the other way around, skip some files that actually are not ARWs)?

Reply:
To be honest, I am not 100% sure that it will cover all the ARW files or skip some files that are not actually ARWs. I have tested with a few of them(ARW extension, TIFF extension, TIFF extension with ARW format) and it works well. I am relying on the exiftool implementation at the moment. I think we would be able to track the first scenario by looking at the Appsignal errors. Not sure how to tackle the second one though